loginCompose/src/main/res/values/themes.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
```

Comment:
This is an xml theme that doesn't make sense to be in a feature module

Reply:
removed xml theme